src/parser.js

```diff
@@ -154,6 +154,11 @@ export default class Parser {
 
   // == Internal properties & functions ==
 
+  // Base path (URL) name for the API
+  get basePath() {
+    return (this.swagger.basePath || '').replace(/[/]+$/, '');
```

Comment:
Is this the correct move? In other words, can we safely assume the user will always put their slashes on the front of the  `href` and not on the end of `basePath`?

- Base Path: `http://example.com/`
- Resource HREF: `users/{id}`